source/localizable/object-model/computed-properties.md

```diff
@@ -19,22 +19,24 @@ Person = Ember.Object.extend({
   })
 });
 
-var ironMan = Person.create({
+let ironMan = Person.create({
   firstName: 'Tony',
   lastName:  'Stark'
 });
 
 ironMan.get('fullName'); // "Tony Stark"
 ```
 
-This declares the function to be a computed property, and the arguments tell Ember that it depends on the `firstName` and `lastName` attributes.
-
-Whenever you access the `fullName` property, this function gets called, and it returns the value of the function, which simply calls `firstName` + `lastName`.
+This declares `fullName` to be a computed property, with `firstName` and `lastName` as the properties it depends on. The
```

Comment:
Every period should be followed by a newline, to make translations and future changes easier.